### ISSUE TITLE
filemanager update

### ISSUE THREAD

**Issue by bqphone** [ubports/filemanager-app#15]
hi there,

using bq 4.5 ubports OS

When i opened openstore today and upgraded it, i got an option to upgrade filemanager, after i did that, i did later went to system settings and checked if any upgrade, so i got another upgrade for filemanager, so i did it, and went back to openstore, the upgrade was again there.

So i guess system and openstore has 2 version of filemanager that keep canceling each other.

Regards,

**Comment by sverzegnassi** [comment]
@NeoTheThird Would you mind to upload the version 0.4.562 on the OpenStore, which is the latest version released on Ubuntu Store and the one which is preinstalled in UBports images?

**Comment by sverzegnassi** [comment]
The issue has been fixed.

**Comment by mihaelmilea** [comment]
Are you sure this was fixed? I mean yes, the "dance" between open store and system settings -> updates stopped but I think that is only because the updated open store version somehow dissapeared and now open store is still serving the old version 0.4.562 which is found in the old Ubuntu store and was released on 1 august 2016. So please help me: Do we have the new updated version in the open store? It doesn't seem to me because there are no new features, the "unlock full access" menu is still there and the version number is the same old one.

**Comment by sverzegnassi** [comment]
@mihaelmilea The new version of File Manager (the one featured on the forum, for instance) hasn't been released yet.

The recent update we have released is the latest package available on the Ubuntu Store (v. 0.4.562)[1], and it's newer than the package that used to be available on OpenStore for a while (v. 0.4.528)[1]. So yes, the issue is fixed. :)

The differences you note (e.g. "unlock full access") weren't actually official features or improvements, since come from a different - yet unknown - branch, which probably only exists locally on some machine of former Canonical employees. Indeed, it was an unofficial package released when OpenStore was just the "geek place" where only apps with restricted permissions were accepted.

[1] From the OpenStore APIs:
`"revisions": [
      {
        "revision": 5,
        "version": "0.4.528",
        "downloads": 1614
      },
      {
        "revision": 6,
        "version": "0.4.562",
        "downloads": 14
      }
    ],`

**Comment by mihaelmilea** [comment]
Thank you for your answer. I guess I was confused. When that versions "dance" happened I could have sworn that the file manager that I installed from the Open Store didn't have the "unlock full access" menu while being able to browse the SD card and root of the file system without entering any root password. And then, because the update message in system settings -> updates was bothering me, I installed it and it downgraded it to the previous version and now I cannot find the newer (or so it seemed to me) version in Open Store.
